samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/main.c

```diff
@@ -105,9 +105,13 @@ static void light_default_status_init(void)
 	}
 
 	default_tt = gen_def_trans_time_srv_user_data.tt;
+
+	target_lightness = lightness;
+	target_temperature = temperature;
 }
 
-void update_light_state(void)
+/* Setting of hardware as per association with Models */
+void set_hardware(void)
```

Comment:
How about update_led_gpio()? I think that's more descriptive than "hardware".

Reply:
Unrelated to this PR, this function would be cleaner if it took two parameters: lightness and temperature, instead of picking them from some global variables. In general I think you have much too much global state in your application.

Reply:
lightness, temperature are global variables because it is part of State Binding algorithm.
And they are not recently introduced.

I have not specially declared them as global so that they could use here.

Reply:
>> How about update_led_gpio()? I think that's more descriptive than "hardware".
Ok. I will do this.

Reply:
done !!